src/schemas/v2/combined.json

```diff
@@ -36814,6 +36814,182 @@
+                "properties": {
+                  "type": {
+                    "type": "string",
+                    "enum": [
+                      "Sale",
+                      "Sold"
+                    ]
+                  },
+                  "date": {
+                    "title": "Date",
+                    "type": "string",
+                    "format": "date-time"
+                  },
+                  "price": {
+                    "title": "Price",
+                    "type": "object",
+                    "properties": {
+                      "amount": {
+                        "title": "Price",
+                        "type": "number"
+                      },
+                      "currency": {
+                        "title": "Currency",
+                        "type": "string",
+                        "enum": [
+                          "USD",
+                          "EUR",
+                          "GBP",
+                          "JPY",
+                          "CAD",
+                          "AUD",
+                          "Other"
+                        ]
+                      }
+                    }
+                  }
+                }
+              },
+              "propertyFullDescription": {
+                "title": "Property description",
+                "type": "string"
+              },
+              "uprn": {
+                "title": "Uprn",
+                "type": "string"
+              },
+              "propertyType": {
+                "title": "Property Type",
+                "type": "string",
+                "enum": [
+                  "Apartment",
+                  "Flat",
+                  "Property",
+                  "Detached House",
+                  "Maisonette",
+                  "Semi-detached House",
+                  "Terraced House",
+                  "Others"
+                ]
+              },
+              "tenure": {
+                "title": "Ownership Type",
+                "description": "What type of ownership is the property?",
+                "type": "string",
+                "enum": [
+                  "Freehold",
+                  "Managed Freehold",
+                  "Commonhold",
+                  "Leasehold",
+                  "Other"
+                ]
+              },
+              "BasicDetails": {
+                "title": "Basic Property Details",
+                "type": "object",
+                "properties": {
+                  "bedrooms": {
+                    "title": "Number of bedrooms",
+                    "type": "integer"
+                  },
+                  "bathrooms": {
+                    "title": "Number of bathrooms",
+                    "type": "integer"
+                  },
+                  "yearBuilt": {
+                    "title": "Built Year",
+                    "type": "string",
+                    "format": "date"
+                  },
+                  "floorArea": {
```

Comment:
We don't need both `inSquareMeter` and `inSquareFoot` - in the main propertyPack schema we have 'internalFloorArea' and should probably align with that.